gemfiles/6.0.gemfile

```diff
@@ -0,0 +1,17 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "sqlite3", "~> 1.4", :platforms => :ruby
```

Comment:
Is the idea here to manually ignore sqlite3's requirement of Ruby version less than 3.0?

Reply:
No, it's because this line is [from the file I copied it from](https://github.com/entretechno/paperclip/blob/15f787ff8c351d0590a2d8bbe2d1bbedbe7f9003/gemfiles/5.0.gemfile#L5), so I updated it to [Rails 6.0's requirement](https://github.com/rails/rails/blob/v6.0.0/Gemfile#L123)

Reply:
Ah, gotcha!